install-components.md

```diff
@@ -160,10 +160,28 @@ that you plan to create the `Workload` in:
     ```
 
 3. Give developers namespace-level access and view access to appropriate cluster-level resources by doing one of the following:
-  * Use the `tanzu auth` plug-in to grant `app-viewer` or `app-editor` roles
-  * Apply the following RBAC policy:
+
+  **Note:** In order to apply the bindings below, an adminstrator account is required. However to use the pre-defined roles, a non adminstrator account (i.e. developer account) is required.
```

Comment:
`In order to apply the bindings below, an adminstrator account is required.` Is this referring to the first option (using plugin) or the second option (applying RBAC policy) that needs an administrator account, or both? 

`However to use the pre-defined roles, a non adminstrator account (i.e. developer account) is required.` this sentence is a bit confusing to me. Do you mean that only a non administrator account can be bound to the pre-defined roles? Also typo in `adminstrator` should be `administrator`